include/base/CardinalEnums.h

```diff
@@ -91,7 +91,10 @@ enum NekWriteEnum
   heat_source,
   x_displacement,
   y_displacement,
-  z_displacement
+  z_displacement,
+  mesh_velocity_x,
+  mesh_velocity_y,
+  mesh_velocity_z
```

Comment:
I see in NekInterface that the names we use to index into the `usrwrk` were renamed, like `displacement_x` because `mesh_velocity_x`. What is the meaning of `x/y/z_displacement` now? Should we be deleting them from `NekWriteEnum`?

Reply:
I am using `mesh_velocity_x/y/z` for mesh velocities because `displacement_x/y/z` have been used elsewhere for variables associated with volume-based displacements that are never written to `usrwrk` - we add them directly to `mesh->x/y/z`, then copy to device. I don't really see `NekWriteEnum::displacement_x/y/z` being used anywhere, so I think we can delete them from `NekWriteEnum`.

Reply:
Actually, we are using `field::x/y/z_displacement` with `writeVolumeSolution` for volumetric mesh displacement. That said, there's no reason for these two to be different, except for clarity around variable names (displacement vs mesh-velocity)

Reply:
Ok I see. This is fine like you have it.